bindings/python/iota_sdk/client/_utils.py

```diff
@@ -36,23 +36,31 @@ def _call_method(self, name, data=None):
         """
 
     # pylint: disable=redefined-builtin
-    def hex_to_bech32(self, hex_str: HexStr, bech32_hrp: str) -> str:
+    def hex_to_bech32(self, hex_str: HexStr, bech32_hrp: Optional[str]) -> str:
```

Comment:
No need to add a default `= None` for all of them like this?
```suggestion
    def hex_to_bech32(self, hex_str: HexStr, bech32_hrp: Optional[str] = None) -> str:
```

Reply:
So users don't have to provide `None` themselves if they don't want to provide a value

Reply:
Ah damn yeah you're right sorry

Reply:
Done